LICENSE.md

```diff
@@ -0,0 +1,13 @@
+Copyright 2017 Michał Muskała
```

Comment:
Let's rather use "The ElixirBench team" - it's not just me 🙂

Reply:
My mistake! Done! :+1: